tests/run-make-fulldeps/obtain-borrowck/driver.rs

```diff
@@ -24,7 +24,7 @@ use rustc_hir::def::DefKind;
 use rustc_hir::def_id::LocalDefId;
 use rustc_interface::interface::Compiler;
 use rustc_interface::{Config, Queries};
-use rustc_middle::query::query_values::mir_borrowck;
+use rustc_middle::query::queries::mir_borrowck::ProvidedValue;
```

Comment:
This should import the actual type from rustc_middle, not go through query type aliases.

Reply:
I think the idea is to avoid relying on the return type, but I'm not sure why.